content/docs/reference/neon-cli.md

```diff
@@ -0,0 +1,160 @@
+
+| Command                                                 | Subcommands                            | Description               |
+|---------------------------------------------------------|----------------------------------------|---------------------------|
+| [auth](../reference/cli-auth)                                     |                                        | Authenticate              |
+| [projects](../reference/cli-projects)                             | `list`, `create`, `update`, `delete`, `get` | Manage projects           |
+| [me](../reference/cli-me)                                         |                                        | Show current user         |
+| [branches](../reference/cli-branches)                             | `list`, `create`, `update`, `delete`, `get` | Manage branches           |
+| [endpoints](../reference/cli-endpoints)                           | `list`, `create`, `update`, `delete`, `get` | Manage endpoints          |
+| [databases](../reference/cli-databases)                           | `list`, `create`, `delete`             | Manage databases          |
+| [roles](../reference/cli-roles)                                   | `list`, `create`,  `delete`            | Manage roles              |
+| [operations](../reference/cli-operations)                         | `list`                                 | Manage operations         |
+| [connection-string](../reference/cli-connection-string)           |                                        | Get connection string     |
+
+## Global options
+
+Global options are supported with any Neon CLI command.
+
+| Option      | Description                         | Type   | Default                           |
+| :---------  | :---------------------------------- | :----- | :-------------------------------- |
+| [--version](#version)   | Show the Neon CLI version number                 | boolean| -                                 |
+| [--help](#help)      | Show the Neon CLI help                           | boolean| -                                 |
+| [-o, --output](#output)| Set the Neon CLI output format                   | string | table                           |
+|             | Possible choices: "json", "yaml", "table" | |                                  |
+| [--api-host](#api-host)  | The API host                        | -      | https://console.neon.tech/api/v2|
+| [--config-dir](#config-dir)| Path to the Neon CLI configuration directory            | string | `/home/<user>/.config/neonctl`   |
+| [--oauth-host](#oauth-host)| URL for the Neon OAuth host              | -      | https://oauth2.neon.tech        |
+| [--client-id](#client-id) | OAuth client id                     | string | neonctl                         |
+| [--api-key](#api-key)   | Neon API key                             | string | ""                                |
+| [--analytics](#analytics) | Enable analytics                    | boolean| true                              |
```

Comment:
I realised that there is not much sense in such description, cause the only purpose of that flag is to give users ability to switch off analytics, so the actual usage would be like this `neonctl --no-analytics`. I updated this option description to:
```
      --analytics   Manage analytics. Example: --no-analytics, --analytics false
                                                       [boolean] [default: true]
```

Reply:
Updated with `--no-analytics, --analytics false` examples, but I am still unsure what analytics is.

Reply:
We gather usage data to analyze CLI usages. Now we collect the command, user id, some options. Of course we don't collect any user defined data, like project ids, creation payloads etc

Reply:
We don't share this data with anyone, we plan to use it to improve user experience